spec/components/action_link_spec.rb

```diff
@@ -171,15 +108,4 @@ def component_name
     )
     assert_select ".gem-c-action-link--light-text"
   end
-
-  it "has data attributes if data attributes are passed in" do
```

Comment:
Overall this is a great bit of cleanup, I'm a bit puzzled by the removal of this test, though? Could it not be rewritten and remain valuable? Or do similar enough tests exist elsewhere to invalidate the need for it?

Reply:
Thanks for all the reviews @maxgds, much appreciated. This test should already be covered by tests in the component wrapper helper, which is now applied to this component (replacing the `data` option with `data_attributes`) see - https://github.com/alphagov/govuk_publishing_components/blob/main/spec/lib/govuk_publishing_components/components/component_wrapper_helper_spec.rb

Reply:
Ahh, makes sense, thanks!

Reply:
You've got me thinking though - while part of the value of the component wrapper is to reduce what are effectively duplicate tests across components, it still might be good to have something that confirms that the component wrapper has been installed correctly and is working, and continues to work. I've got some rough ideas, I'll put together a PR soon.